documentation/modules/ROOT/pages/transformations/header-to-value.adoc

```diff
@@ -0,0 +1,154 @@
+:page-aliases: configuration/header-to-value.adoc
+// Category: debezium-using
+// Type: assembly
+// ModuleID: header-to-value
+// Title: Moves or copies headers into the record value
+[id="header-to-value"]
+= HeaderToValue
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+This SMT permits to copy or move headers to record value. Supports multiple headers and also nesting on the value through dot notation.
```

Comment:
@roldanbob 

> can use dot notation to generate message values that are composed of nested header fields.

The dot notation on the field option permits to specify the place in the record value where you want to put the header. For example if you have `header1` you can configure the field `source.header1` to say that the header must go inside the source structure. If you use only `header1` as field configuration it will be put in the 'root` of the payload struct.